ladybug/datacollection.py

```diff
@@ -163,13 +163,13 @@ def get_highest_values(data, number_of_highest_values):
         """
         lenght_data = len(data)
         
-        highest_values_index = sorted(range(lenght_data), key = lambda k: data[k], reverse = True)[0:number_of_highest_values]
+        highest_values_index = sorted(range(lenght_data), key = lambda k: data[k], reverse = True)[0:int(number_of_highest_values)]
         
         is_data_in_highest_values = [0]*lenght_data 
-        for i in range(number_of_highest_values):
+        for i in range(int(number_of_highest_values)):
             is_data_in_highest_values[highest_values_index[i]]=1
             
-        highest_values = sorted(data, reverse = True)[0:number_of_highest_values]
+        highest_values = sorted(data, reverse = True)[0:int(number_of_highest_values)]
```

Comment:
What if you add `number_of_highest_values = int(number_of_highest_values)` once on the top and then use the integer value?